### ISSUE TITLE
Introduction of community ladder

### ISSUE THREAD

**Issue by tomkerkhove** [kedacore/governance#30]
As KEDA grows, so does its community and we should consider introducing a community ladder to give credit to those who keep on contributing day-in, day-out (regardless of contribution type).

There are a few options we can consider:
- Formal Community Membership (similar to [Dapr's approach](https://github.com/dapr/community/blob/master/community-membership.md#community-membership))
- Introduce KEDA badges (similar to [Linkerd Hero](https://www.youracclaim.com/badges/538d249f-ec6d-4c5c-93f4-44d7c5596b36/twitter) thanks to [CNCF service desk](https://github.com/cncf/servicedesk))

This idea was initiated due to https://github.com/kedacore/governance/discussions/29 which is a valid question.

**Comment by tomkerkhove** [comment]
I'm personally a fan of the KEDA badges as it's an easy way to start recognizing hard work by the community without introducing too much of hierarchy.

**Comment by tomkerkhove** [comment]
Any thoughts @zroubalik @jeffhollan @ahmelsayed?

**Comment by zroubalik** [comment]
I like a simple "formal" approach, having regular contributors in GitHub Group allows us to assing these users to issues etc. But I am open to badges as well :)

**Comment by tomkerkhove** [comment]
Created https://github.com/kedacore/governance/issues/32

**Comment by JorTurFer** [comment]
Should we push this topic @kedacore/keda-maintainers ? The adoption is growing there are some new people contributing, maybe if we move this topic, they will be more engaged and open to continue contributing 😄

**Comment by Ritikaa96** [comment]
Hi! is there any progress on this one? are we still looking for badges or memberships?

**Comment by JorTurFer** [comment]
IMO, badges look really nice, and it's a thing that contributors can share with others (with proud I hope). Maybe create a team inside the org and invite them to the org, could be another good option to improve the feeling of being part of KEDA.
At the end of the day, common contributors has the engagement and the "influence" in the project without the badges or another "candy", but for new contributors it could be a good option.
We should define in the proper way the needed steps in each case, of course.

But, as you said @tomkerkhove , I'd like to hear the opinion from @Ritikaa96 and maybe we can open a thread in Slack to gather people opinions too

**Comment by tomkerkhove** [comment]
> But, as you said @tomkerkhove , I'd like to hear the opinion from @Ritikaa96 and maybe we can open a thread in Slack to gather people opinions too

Let's ask people to share things here to keep things in one place, maybe?

**Comment by zroubalik** [comment]
A t-shirt for a major contribution? 😄

**Comment by zroubalik** [comment]
I would go with both badges and a group, for example to join a group you need to have at least XYZ commits, or something around this. The same for badges.

 I think that it is great to recognize contributors.

**Comment by JorTurFer** [comment]
I think that using both, we have more flexibility. Maybe we can have 2-3 badges (IDK if it's possible) and also the groups.
Let me share this issue in #keda-dev slack channel (referring to this issue).
KEDA is growing and recognition and the feeling of being part of KEDA it's necessary to grow not only in users but also with contributors

**Comment by Ritikaa96** [comment]
Hi @JorTurFer @tomkerkhove @zroubalik, the badges are what am looking forward to. but with that how about a scoring system or different badges for a number of contributions to the team. It will make the contributing process fun if we allocate scores to their activity and more people would like to engage. The process can look like this:
1. Badges vary with number & quality of contribution
2. A hierarchy in the scoring system, just like stages in a game provides.
3. weekly or monthly shout out to whoever reached certain milestones in the meeting. It's not a hassle but it will surely provide encouragement to new contributors. 

I have yet to find an open-source project supporting this system or is it even feasible, but it'll be highly encouraging to participate in this process.

**Comment by tomkerkhove** [comment]
Just dropping a note that I haven't forgotten about this thread and will circle back.

I think introducing badges is a simple way to recognize people's contribution, but it would need clear requirements. The simplest example would be "Scaler Contributor" for everyone who contributes a scaler along with docs and has been around to maintain it for x months.

This is something simple that I can have a look at as a first step. Thoughts @zroubalik @JorTurFer?

**Comment by JorTurFer** [comment]
I agree. Maybe we can create other categories like yours, IDK, something like:
- Scaler Contributor
	- They have been increasing/improving scaler catalogue (with the proper docs) for X months
- Chart Contributor
	- They have been improving helm charts (with the proper docs) for X months
- Global Contributor
	- They have been contributing everywhere for X months
- HTTP Add-on Contributor ???
	- They have been contributing HTTP-Add-on for X months

**Comment by tomkerkhove** [comment]
I created https://github.com/kedacore/governance/issues/58 to introduce our first badge and see how we proceed based on the learnings.

**Comment by tomkerkhove** [comment]
Another aspect to clarify/doc - Who can trigger e2e tests and how/when do people get access to that.

ie. "core contributor"

**Comment by tomkerkhove** [comment]
@Ritikaa96 Is this what you are looking for?

https://github.com/dapr/community/blob/master/community-membership.md

**Comment by tomkerkhove** [comment]
Still working on this though, just behind the scenes.

**Comment by tomkerkhove** [comment]
Apart from the badge, I think we need a stronger outcome here. Thoughts @zroubalik / @JorTurFer ?

**Comment by zroubalik** [comment]
Agree!

**Comment by JorTurFer** [comment]
Honestly, IDK what more we could give the contributors more than badges and adding them to the org if they are recurrent. I mean, we don't have any bounty budget or so

**Comment by JorTurFer** [comment]
I think that these new role proposal can be considered too as community recognition https://github.com/kedacore/governance/issues/107
